docs/man/ctags-lang-verilog.7.rst

```diff
@@ -82,7 +82,7 @@ because they cannot be overridden.
 "output.tags"
 with "--options=NONE --sort=no -o - --fields-SystemVerilog=+{parameter} input.sv"
 
-.. code-block:: tags
+.. code-block:: ctags
```

Comment:
"tags" is specified to highlight the code block as a tags file.
Though there is no highlighter for tags file in pygments, I would like to keep "tags" as is.
Specifying "tags" in `.. code-block:: tags` may make sense when someone implements a highlighter for tags file.
See https://github.com/pygments/pygments and https://pygments.org/languages/. 

"ctags" is not good name for specifying highlighter for tags file.
I would like to implement or I would like to see someone implements a highlighter for `.ctags` file like optlib/RSpec.ctags.

These two are different.
So exchanging tags and ctags in `.. code-block:: ` is unacceptable. 

If you want to remove "tags" and/or  "ctags" because they are not implemented in the current pygments, it is acceptable.

Reply:
I see.
I changed to ctags, because I saw warning messages showing ctags were used in news.rst.  But they were for optlib type code blocks.
I've reverted this change.